types/aws-lambda/tsconfig.json

```diff
@@ -26,6 +26,7 @@
         "test/cloudwatch-tests.ts",
         "test/codepipeline-tests.ts",
         "test/codebuild-tests.ts",
+        "test/connect-contact-flow-tests.ts",
```

Comment:
Hmm that's annoying, it seems adding tests triggers the config changed label so you don't get the nice new automated merge flow. They should probably fix that....